front-end/src/components/Login/Forgot/index.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import './index.scss';
+import $ from 'jquery';
+
+class Forgot extends Component{
+
+    componentDidMount = () => {
+        $('.forgot__close').on('click', function () {
+            $('.forgot').fadeOut();
+/*
+            $('.login').fadeIn();
+*/
+        });
+    };
+
+    render(){
+        return (
+
+            <div className="forgot">
+{/*
```

Comment:
Commented code

Reply:
@Thealoner  Ok, now how i can change this if i already commited it? Delete this request and create new one?

Reply:
Just edit/remove the lines that have the comments and commit the changes.